### DISCUSSION TITLE
Align label and range to be in the same line

### DISCUSSION THREAD

**Discussion by idhamari** [twbs/bootstrap#36765] [General] [answered]
Dear all, 
when using the bootstrap, it messes the range input, now the label and the range are sparated in different lines. e.g.

 
          <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.2.0/css/bootstrap.min.css">
          <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/bootstrap-select/1.8.1/css/bootstrap-select.css">
          <script src="https://cdnjs.cloudflare.com/ajax/libs/bootstrap-select/1.8.1/js/bootstrap-select.js"></script>
          <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.2.0/js/bootstrap.min.js"></script>

       <div>
        <label for="zValue"> Z data value:</label> 
       <input type="range" id="zValue" name="zValue" value="0" min="0" max="100" step="0.1" oninput="this.nextElementSibling.value = this.value" style="width: 100px;">
       <output>0</output>     
      </div>

Gives this output

<img width="115" alt="bs01" src="https://user-images.githubusercontent.com/28802516/179485125-f1fed81f-ebf9-4b36-96c2-0471fc727157.PNG">

Any suggestion to solve this?
best

e.g.

**Comment by TravelTimN** [comment, marked as answer]
@idhamari 
Bear in mind, that you've got Bootstrap version 3.2, which was released almost a decade ago (June 2014), and there are a lot more flexible options using the more recent versions of Bootstrap. That being said, version 3.2 isn't even listed anymore as a valid version.
[see list of maintained Bootstrap versions](https://getbootstrap.com/docs/versions/)

If possible for your application, I'd recommend using Bootstrap v.5 for example. A nice example below can be seen using the Bootstrap [input-groups](https://getbootstrap.com/docs/5.2/forms/input-group/).

HTML
```html
<link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.0-beta1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-0evHe/X+R7YkIZDRvuzKMRqM+OrBnVFBL6DOitfPri4tjfHxaWutUpFmBp4vmVor" crossorigin="anonymous">
<script src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.0-beta1/dist/js/bootstrap.bundle.min.js" integrity="sha384-pprn3073KE6tl6bjs2QrFaJGz5/SUsLqktiwsUTF55Jfv3qYSDhgCecCxMW52nD2" crossorigin="anonymous"></script>

<div class="input-group me-5 w-25">
  <span class="input-group-text" id="zValueLabel">Z data value:</span>
  <input type="range" class="form-range form-control me-3 px-1" id="zValue" name="zValue" value="0" min="0" max="100" step="0.1" oninput="this.nextElementSibling.value = this.value">
<output>0</output>
</div>
```

CSS
```css
input[type="range"] {
  height: 38px;
}

.form-control:focus {
  box-shadow: none;
}

output {
  font-size: 1.5em;
}
```

Preview:
![image](https://user-images.githubusercontent.com/41592028/179700278-4218563f-a7bf-4094-9594-e72de27d3503.png)

I hope you found this helpful to proceed with your project!

**Comment by idhamari** [comment]
@TravelTimN Thanks for your suggestion, I will try that!

**Comment by idhamari** [comment]
It works, thanks!

**Reply by TravelTimN** [reply]
Excellent! Glad to have helped!

Remember, if you're going to use that version of Bootstrap, you need to only use those components. Syntax between version 3 and 5 have completely changed, so if you use version 5 components with version 3 CDN links, then it won't work. The data attributes have `-bs-` between element names now.
